mantis-control-plane/mantis-control-plane-dynamodb/src/main/java/io/mantisrx/extensions/dynamodb/DynamoDBLeaderElector.java

```diff
@@ -139,6 +139,7 @@ protected boolean tryToBecomeLeader() {
                                     .withSessionMonitor(safeTimeWithoutHeartbeatMs, Optional.of(this::giveUpLeadership))
                                     .build());
             if (optionalLock.isPresent()) {
+                log.info("became leader");
```

Comment:
nit: consider adding leader identity like ip-address, etc.
I'm guessing those are already logged as part of the metadata - MDC/NDC tags